### PR TITLE
correct the location where to import validate_api_key() and also comments

### PR DESCRIPTION
**Pull request by bendesu**
correct the location of importing validate_api_key()

### CODE REVIEW

File: jovian/jovian_nb_ext/main.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ define([
         };
 
         const validate_api =
-          "from jovian.utils.api import validate_api_key\n" +
```
Comment (sidujjain) [inline jovian/jovian_nb_ext/main.js]: Please update this in comments above

Comment (bendesu) [reply jovian/jovian_nb_ext/main.js]: One commend updated.

**Comment by sidujjain** [comment]
I've reverted whitespace changes. Try to avoid unwanted changes. 

Also avoid adding `()` in branch names.

**Comment by bendesu** [comment]
Got it! Sorry, the whitespace change I believe that its automatically done by the vs code editor. I will try to avoid them by setting the vs code and also the branch name with (). Thank you! :)

**Comment by sidujjain** [comment]
np. you can install `prettier` extension, so that our formatting is consistent. 

RE `()`: bash treats branch name as function when you try to checkout the branch.

**Comment by bendesu** [comment]
Sure, I will install the prettier now, and I will keep checking editor as well.

I see, I will avoid using name with () in future. Thank you for helping me :)

**Pull request merged**